recipe/meta.yaml

```diff
@@ -10,11 +10,9 @@ source:
   sha256: 0b78abeb264d1810a245e563964f1b5d9f380afae75fff032b4499cf300598be
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  entry_points:
-    - wily = wily:main
```

Comment:
why is the entry point removed? Won't this mean the `wily` command isn't registered?

Reply:
If the entrypoint is properly configured for PyPI, setting this information here as well will only cause issues. With the current build of the package, I receive the following error:
```
(wily_env) ~ >>> wily                                                                  
Traceback (most recent call last):
  File "/home/me/miniconda3/envs/wily_env/bin/wily", line 6, in <module>
    from wily import main
ImportError: cannot import name 'main' from 'wily' (/home/me/miniconda3/envs/wily_env/lib/python3.9/site-packages/wily/__init__.py)
```

I have added a smoke test below to ensure the CLI is working properly when the entrypoint configuration is removed (and it is). I think no further changes are needed!